WalletWasabi.Fluent/TreeDataGrid/TreeDataGridPrivacyTextCell.cs

```diff
@@ -38,37 +38,30 @@ public override void Realize(IElementFactory factory, ICell model, int columnInd
 
 	public override void Render(DrawingContext context)
 	{
-		if (_formattedText is not null)
+		if (_formattedText is null)
 		{
-			var r = Bounds.CenterRect(_formattedText.Bounds);
-			context.DrawText(Foreground, new Point(0, r.Position.Y), _formattedText);
+			return;
 		}
+
+		var r = Bounds.CenterRect(_formattedText.Bounds);
+		context.DrawText(Foreground, new Point(0, r.Position.Y), _formattedText);
 	}
 
 	protected override void OnAttachedToVisualTree(VisualTreeAttachmentEventArgs e)
 	{
-		base.OnAttachedToVisualTree(e);
-
-		if (Realized.Count == 0)
-		{
-			Subscription = Services.UiConfig
-				.WhenAnyValue(x => x.PrivacyMode)
-				.ObserveOn(RxApp.MainThreadScheduler)
-				.Subscribe(x => SetContentVisible(!x));
-		}
-
-		Realized.Add(this);
+		var displayContent = ObservableMixin.DelayedRevealAndHide(
+			this.WhenAnyValue(x => x.IsPointerOver),
+			Services.UiConfig.WhenAnyValue(x => x.PrivacyMode));
```

Comment:
Okay, I see now why you put the logic into `ObservableMixin` but the class name is not correct then. It should be `PrivacyModeHelper` or something.